test/Microsoft.Framework.Runtime.FunctionalTests/ServiceBreadcrumbsFacts.cs

```diff
@@ -34,7 +35,8 @@ public void BreadcrumbsAreCreatedSuccessfully()
             Assert.True(File.Exists(Path.Combine(TempFolderPath, "Test.1.0.0")));
         }
 
-        [Fact]
+        [ConditionalFact]
+        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
```

Comment:
Servicing is not available on *Nix. We don't need this test to run because we don't write the breadcrumbs there. Also, I'm not sure that this test will pass even if you get the user name because it tries to set access rules on folders (does that work on Mono?)